examples/vitess/vtctld-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: vtctld
+  labels:
+    name: vtctld
+spec:
+  ports:
+    - port: 15000
+      targetPort: 15000
+      nodePort: 30000
```

Comment:
do we have to hardcode 3000? other 2 are ok but nodePort needs to be available on all nodes in the cluster, so its a little awkward.